wiki/Beatmaps/Patterns/Stream/en.md

```diff
@@ -0,0 +1,12 @@
+---
+tags:
+  - streams
+---
+
+# Stream
+
+A **stream** is an [osu!](/wiki/Game_Modes/osu!) gameplay element where consecutive circles are grouped together and intended to be played by alternating clicking between fingers. Streams are usually played with smooth movement, as opposed to a snappy movement of [jumps](/wiki/Beatmaps/Patterns/Jump).
+
+In most beatmaps, streams represent high note density in a song and are snapped to 1/4 ticks on the [hitobjects timeline](/wiki/Beatmap_Editor/Timelines#hit-objects).
```

Comment:
shouldnt it be like "Most streams represent ..." not most beatmaps
same for jump artcile